x-pack/test/performance/journeys/base.config.ts

```diff
@@ -42,6 +42,7 @@ export default async function ({ readConfigFile, log }: FtrConfigProviderContext
     ...(prId !== undefined ? { prId } : {}),
     testJobId,
     testBuildId,
+    isBareMetal: process.env.BARE_METAL_MACHINE === 'true',
```

Comment:
Do you think that adding `ciBuildName` would be another option for this? Then we could filter by `kibana-single-user-performance` and I think that would make more sense to people querying this data.

Reply:
Yeah, definitely! but I think it should be better as an extra field. I am kinda feeling these env variables, labels, configs etc. sometimes carrying more meaning than the actual word meaning and as a result we end up having conditional logics which gives mental overhead while reading the code. So my pref. would be being explicit with the field. Of course we can understand by simply saying:
`isBareMetal : ciBuildName === "kibana-single-user-performance"` but in that case we assume that the person knows we only run bareMetal in kibana-single-user-performance builds.

Reply:
Cool, I assumed that people were going to mostly be using this flag when querying the data in the dashboard, and in that case I figured it was less mental gymnastics. Sounds like that's not the case and what we really care about is exactly "is the machine using bare metal". Why do we care about that info specifically?

Reply:
I think the my answer to that is we can have different steps running on different [hardware](https://github.com/suchcodemuchwow/kibana/blob/KO-739-add-bare-metal-machine-label-to-performance-telemetry-data/.buildkite/pipelines/performance/daily.yml). My assumption is that it's the most specific field we can target for performance tests running on _[bare metal machine](https://github.com/suchcodemuchwow/kibana/blob/KO-739-add-bare-metal-machine-label-to-performance-telemetry-data/.buildkite/pipelines/performance/daily.yml#L15) on hourly pipeline_

Reply:
I think I understood what you meant more clearly now which is not having `isBareMetal` field at all but only adding `ciBuildName`. I thought you meant having isBareMetal field which is continually tied to ciBuildName. However my only concern is that let's say for some reason we have repeated performance test steps in this pipeline but only on different hardware, how are we going to understand that it's running on bare metal machine ? I assume the cleanest option in that case having this isBareMetal in this [line](https://github.com/suchcodemuchwow/kibana/blob/KO-739-add-bare-metal-machine-label-to-performance-telemetry-data/.buildkite/pipelines/performance/daily.yml#L20) :
```yaml
env:
  BARE_METAL_MACHINE: 'true'
```

Reply:
I believe right now scalability tests are also reporting to EBT cluster because they are pretty much based on single user performance test configs.

Reply:
> how are we going to understand that it's running on bare metal machine ?

I assumed that wasn't a question we would really ask about this data but I seem to be wrong.